CMakeLists.txt

```diff
@@ -1,88 +1,44 @@
-cmake_minimum_required(VERSION 3.11)
-if(COMMAND cmake_policy)
-	cmake_policy(SET CMP0003 NEW)
-	cmake_policy(SET CMP0005 NEW)
-endif(COMMAND cmake_policy)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.11)
+IF(COMMAND cmake_policy)
+	CMAKE_POLICY(SET CMP0003 NEW)
+	CMAKE_POLICY(SET CMP0005 NEW)
+ENDIF(COMMAND cmake_policy)
```

Comment:
Are these policy statements still needed? We have a much more recent minimum version now, so perhaps they're the default.